### ISSUE TITLE
Master Admins ban

### ISSUE THREAD

**Issue by 1234414** [haxball/haxball-issues#1271]
When master admis ban a player, they are banned. How do I make them not banned?

var banPunishment = false;

 if (ban && byPlayer !== null && byPlayer.id !== 0) {
        room.clearBan(kickedPlayer.id);
        if (byPlayer.id !== kickedPlayer.id) room.kickPlayer(byPlayer.id, "Banning players is not allowed !", banPunishment);
    }
}

**Comment by thenorthstar** [comment]
Duplicate of #1236. Close this issue and don't duplicate old issues more. Thanks.

**Comment by 1234414** [comment]
Not like that. No one in the room can ban player. I want master admins to be able to ban other players but regular admins can't. @thenorthstar

**Comment by 1234414** [comment]
There is a code out there that normal administrators cannot ban main managers.
@thenorthstar

**Comment by 1234414** [comment]
There is a code out there that normal administrators cannot ban main managers.
@thenorthstar

**Comment by thenorthstar** [comment]
> Not like that. No one in the room can ban player. I want master admins to be able to ban other players but regular admins can't. @thenorthstar

@1234414 OK. I understood but the current API of haxball unfortunately cannot protect the players to be banned. I think that you want such a code. Is that it?

**Comment by 1234414** [comment]
No. When normal admins ban any player, they get banned. When you ban any player in Master Admins, it gets banned. I want master admins to be able to ban players.

**Comment by 1234414** [comment]
@thenorthstar

**Comment by 1234414** [comment]
if (ban && byPlayer !== null && byPlayer.id !== 0) {
        room.clearBan(kickedPlayer.id);
        if (byPlayer.id !== kickedPlayer.id) room.kickPlayer(byPlayer.id, "Banning players is not allowed !", banPunishment);
    }
}

**Comment by Abdullah642150** [comment]
The code I threw above is valid for all admins. But I just want it to be valid for regular admins.

**Comment by 1234414** [comment]
The code I threw above is valid for all admins. But I just want it to be valid for regular admins.

**Comment by Abdullah642150** [comment]
I am waiting answer.

**Comment by 1234414** [comment]
I am waiting answer.

**Comment by 1234414** [comment]
If you have a discord can we talk from there please?
